WcaOnRails/app/models/user.rb

```diff
@@ -406,6 +406,10 @@ def can_edit_delegate_report?(delegate_report)
     can_admin_results? || (competition.delegates.include?(self) && !delegate_report.posted?)
   end
 
+  def can_see_admin_competitions?
+    board_member? || senior_delegate?
+  end
```

Comment:
Well, maybe the name was badly thought, but I'm not sure admins would find this information useful...

Giving access to everything just because they're admins may not be necessary.